bool initializeLandmark
float64 angle
---
bool success
geometry_msgs/Point base_location
